istio-telemetry/mixer-telemetry/values.yaml

```diff
@@ -27,6 +27,8 @@ mixer:
         cpu: 4800m
         memory: 4G
 
+    useMCP: true
```

Comment:
And we must stop adding uncommented settings ( and in general add settings only if there are strong use cases for them, it is already far too much )